clipper_admin/clipper_manager.py

```diff
@@ -1031,6 +1047,44 @@ def set_model_version(self, model_name, model_version, num_containers=0):
         for r in range(num_containers):
             self.add_container(model_name, model_version)
 
+    def remove_inactive_containers(self, model_name):
+        # TODO: Test this function
```

Comment:
Yeah I want to add a test for this. Let's get #187 merged before this. Leaving the TODO in for now.